.github/workflows/ci.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     steps:
       - uses: actions/checkout@master
 
-      - name: Set Node.js 12.x
+      - name: Set Node.js 23.x
```

Comment:
Better to stick with even node numbers, ie 22, 23 are the unstable releases which do not stay in support for long, even numbers are LTS versions.

Reply:
Good point. I've brought it up to 24. I've also added a .tool-versions file which is hopefully ok. .tool-versions is used by asdf to manage version of node, python, ruby etc on developer machines. Currently it just specifies the nodejs version and sets it to 24.3.0.